### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toniedzwiedz**
- Add changelog for version 3.0.0 and above to README.md
  - Replace an image with outdated Cognifide and Apache branding with a
  new one

### CODE REVIEW

**Comment by toniedzwiedz** [comment]
Thanks for releasing 3.1.2! People were asking about the change log so I took the liberty of retroactively filling it in in the project README.

Similar notes can be placed on Github in the releases section (kind of like here https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases) but that's not something I could edit.

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ In this case, component is replaced with include tag, and `nocache` selector is
     
 If the filter gets request with selector it'll pass it (using `doChain`) further without taking any action.
 
-![Plain include](https://raw.github.com/Cognifide/Sling-Dynamic-Include/master/src/main/doc/plain-include.png)
+![Plain include](https://raw.github.com/Cognifide/Sling-Dynamic-Include/master/src/main/doc/plain_include.png)
```
Comment (toniedzwiedz) [inline README.md]: Note to self, update this

**Comment by rombert** [comment]
Merged, thanks @toniedzwiedz ! I think in the future we should move this information into the Sling site, but for now this is a good start :-)

**Comment by rombert** [comment]
@toniedzwiedz - I unfortunately had to revert the change due to a build failure:

```
1 Unknown Licenses

*******************************

Unapproved licenses:

  src/main/doc/plain_include.svg

*******************************
```

This can be reproduced using `mvn verify` locally. Can you please submit an updated PR which includes a fix?

**Comment by toniedzwiedz** [comment]
Ah, it didn't occur to me to run the build after adding an image. Apparently the plugin treats the SVG as code. Raised #8

**Comment by rombert** [comment]
RIght, I was suprised as well to see that. 

For the website, the goal is to eventually move the documentation from the README to the page at https://sling.apache.org/documentation/bundles/dynamic-includes.html . If have the time to look into that, our website is hosted at https://github.com/apache/sling-site .

**Pull request merged**